CMake/external_pybind11.cmake

```diff
@@ -75,8 +81,45 @@ function(get_pybind11)
 
 endfunction()
 
+
+# We also want a json-compatible pybind interface:
+function(get_pybind11_json)
+
+    message( STATUS #CHECK_START
+        "Fetching pybind11_json..." )
+    #list( APPEND CMAKE_MESSAGE_INDENT "  " )  # Indent outputs
+
+    # We want to clone the pybind repo and build it here, during configuration, so we can use it.
+    # But ExternalProject_add is limited in that it only does its magic during build.
+    # This is possible in CMake 3.12+ with FetchContent and FetchContent_MakeAvailable in 3.14+ (meaning Ubuntu 20)
+    # but we need to adhere to CMake 3.10 (Ubuntu 18).
+    # So instead, we invoke a new CMake project just to download pybind:
+    configure_file( CMake/pybind11-json-download.cmake.in
```

Comment:
Please integrate this fix:
https://github.com/IntelRealSense/librealsense/pull/11959/files
We need to build in linux and see we didn't ruin something here
Fix was already approved by the user

Reply:
Nir, this is the `pybind11-json` download, not `pybind11`... the changes you pointed to are in the file. Everything is as it should be.

Reply:
My Mistake, :)